app/models/hmpps_api/offender_base.rb

```diff
@@ -8,8 +8,8 @@ class OffenderBase
              :post_recall_release_date, :earliest_release_date,
              to: :sentence
 
-    delegate :indeterminate_sentence?, :immigration_case?,
-             to: :@sentence_type
+    delegate :indeterminate_sentence?, :immigration_case?, :civil_sentence?, :describe_sentence,
+             to: :@sentence
```

Comment:
Can this `delegate to: :@sentence` be combined with the `delegate to: :sentence` directly above it?

Looks like they're both delegating to the same object, since `sentence` is an attr_reader for `@sentence`.